index.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 const path = require('path');
-const trimRepeated = require('trim-repeated');
 const filenameReservedRegex = require('filename-reserved-regex');
-const stripOuter = require('strip-outer');
 
 // Doesn't make sense to have longer filenames
 const MAX_FILENAME_LENGTH = 100;
 
+const reRepeatedReservedChars = /([<>:"/\\|?*\x00-\x1F]){2,}/g; // eslint-disable-line no-control-regex, unicorn/no-hex-escape
+const reOuterReservedChars = /^[<>:"/\\|?*\x00-\x1F]|[<>:"/\\|?*\x00-\x1F]$/g; // eslint-disable-line no-control-regex, unicorn/no-hex-escape
```

Comment:
Can you use the unicode escapes like the below regex?

These also needs to be moved inside the main function as `g` regexes are mutable.